website/server/controllers/api-v3/hall.js

```diff
@@ -11,17 +11,108 @@ import _ from 'lodash';
  * Contributors of tier 8 or higher can use this route.
  */
 
+/**
+ * @apiDefine NoAuthHeaders Missing authentication headers
```

Comment:
I guess they could be moved to `website/server/api-doc.js`. The file is almost empty, but the comment hints that it might be the right place. What do you think?